pkgs/applications/networking/cawbird/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, glib, gtk3, json-glib, sqlite, libsoup, gettext, vala
+, meson, ninja, pkgconfig, gnome3, gst_all_1, wrapGAppsHook, gobject-introspection
+, glib-networking, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.1";
+  pname = "cawbird";
+
+  src = fetchFromGitHub {
+    owner = "IBBoard";
+    repo = "cawbird";
+    rev = "v${version}";
+    sha256 = "sha256:0bk33fh32nnv6ya6j0ij34abw6a3g6m8fq13303slhhja8xhvmb1";
+  };
+
+  nativeBuildInputs = [
+    meson ninja vala pkgconfig wrapGAppsHook python3
+    gobject-introspection # for setup hook
+  ];
+
+  buildInputs = [
+    glib gtk3 json-glib sqlite libsoup gettext gnome3.dconf gnome3.gspell glib-networking
+  ] ++ (with gst_all_1; [ gstreamer gst-plugins-base gst-plugins-bad (gst-plugins-good.override { gtkSupport = true; }) gst-libav ]);
+
+  postPatch = ''
```

you might be able to simplify this with a prePatch step:
```suggestion
  prePatch = ''
    chmod +x data/meson_post_install.py # patchShebangs requires executable file
  '';
```

---

so patchShebangs is already part of the normal patchPhase?

---

I checked, this simplification does not work.

---

:( sorry mate